Microsoft.Azure.CosmosRepository/src/Paging/PageExtended.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) IEvangelist. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.CosmosRepository.Paging
+{
+    /// <inheritdoc/>
+    public class PageExtended<T> : Page<T>, IPageExtended<T> where T : IItem
```

Comment:
I'm not sure I like the name `PageExtended`, it's not really descriptive. What do you think about `PageQueryResult` or something like that? The interface would have to change too.